src/frontend/delete-github-subscription.ts

```diff
@@ -1,16 +1,14 @@
 import { Subscription } from "../models";
 import { Request, Response } from "express";
 
-/**
- * Handle the when a user adds a repo to this installation
- */
 export default async (req: Request, res: Response): Promise<void> => {
-	if (!req.session.githubToken) {
+	const { github, githubToken, jiraHost } = res.locals;
```

Comment:
I think this should be feature flagged, shouldn't it?

Reply:
we can, but it's a lot of trouble.  And currently, this is a bug stopping some people from configuring their github orgs